### ISSUE TITLE
[Feature request] Display list of known signatures on system change

### ISSUE THREAD

**Issue by graywolf** [exodus4d/pathfinder#284]
I would like an option to automatically select system I jump into as current one. Yes, I know pathfinder is able to track my location, but it doesn't select the system I jump into.

By "selecting system" I mean the pretty windows below map showing kills in the system, known signatures etc.

Would it be possible to implement something like that?

**Comment by atkinsj** [comment]
Seconding that this would be a handy usability feature.

**Comment by exodus4d** [comment]
#### I agree it would be a nice feature, but I have a problem with this issue:
- Let us assume that you have set up an autopilot with several jumps.
- On each stargate jump, the new system will be selected automatically and all information panels get reloaded with the current system data
- You will no longer be able to *browse* the map, checking other systems, add system descriptions to other systems, fix/delete signatures in systems you are not in.
- In the worst case, you are currently writing a large system description text to system A, while your client jumps to system B. The changes you already made before saving them to system A, will be discarded and system B panels will load.

I think this can become very frustrating.... So either we have to check if a user is currently interacting with a panel and prevent "system switch", or we make it configurable for users. What you think?

Btw. #281 fixed the bug, where current location link is broken in current release.

**Comment by dkrotil** [comment]
Hi,
even these problems corpmates asking for this, they were happy with Siggy which actively "showed" they position on map.

**Comment by graywolf** [comment]
> we make it configurable for users

Yep, this would be a must. But you actually already have switch in right top corner (I think) for on/off to track current location. You could just add second same switch next to it to on/off "autofocus system on jump" or somesthing like that :)

I agree that if user cannot turn this on/off it could get annoying quickly.
